src/consensus/params.cpp

```diff
@@ -218,12 +258,113 @@ namespace Consensus {
         vFundingStreams[idx] = FundingStream::ParseFundingStream(*this, keyConstants, startHeight, endHeight, strAddresses);
     };
 
-    FundingStreamAddress FundingStream::RecipientAddress(const Consensus::Params& params, int nHeight) const
+    void Params::AddZIP207LockboxStream(
+        const KeyConstants& keyConstants,
+        FundingStreamIndex idx,
+        int startHeight,
+        int addressChangeInterval,
+        int intervalCount)
+    {
+        std::vector<FundingStreamRecipient> recipients(intervalCount, Lockbox());
```

Comment:
```suggestion
        FundingStreamRecipient recipient = Lockbox();
```

Reply:
This isn't a correct change; we retain the timing of the existing funding stream cycle.

Reply:
Okay, but why is it ever necessary to have more than one interval in a lockbox stream?

Reply:
This is mostly to reduce the implementation complexity; this way we don't have to rework all of the logic related to how the value of the funding stream is computed and where it goes at each height (the implementation expects a vector of recipients that we can index into.)